pkg/operator/ceph/file/mds/config.go

```diff
@@ -71,7 +71,10 @@ func (c *Cluster) setDefaultFlagsMonConfigStore(mdsID string) error {
 	configOptions := make(map[string]string)
 
 	// Set mds cache memory limit to the best appropriate value
-	if !c.fs.Spec.MetadataServer.Resources.Limits.Memory().IsZero() {
+	if !c.fs.Spec.MetadataServer.Resources.Requests.Memory().IsZero() {
+		mdsCacheMemoryRequest := float64(c.fs.Spec.MetadataServer.Resources.Requests.Memory().Value()) * mdsCacheMemoryLimitFactor
```

Comment:
Should we rename `mdsCacheMemoryLimitFactor` too? To something more generic since it's used for both.

Reply:
changed to `mdsCacheMemoryFactor`